spec/CloudCode.spec.js

```diff
@@ -1598,6 +1598,32 @@ describe('Cloud Code', () => {
     expect(obj.get('count')).toBe(0);
   });
 
+  it('Pointer should not be cleared by triggers', async () => {
```

Comment:
First word of test descr. is always lowercase; kind of arbitrary, but let's stick to it